packages/blade/src/utils/useInteraction.ts

```diff
@@ -1,7 +1,7 @@
 import type { Dispatch, SetStateAction } from 'react';
 import { useState } from 'react';
 
-export type ActionStates = 'default' | 'hover' | 'focus';
+export type ActionStates = 'default' | 'hover' | 'focus' | 'active' | 'disabled';
```

Comment:
I'm not sure why the diff is showing `active` here. I've removed it. Even when I try to edit this file on github, it doesn't show active. Seems like a bug on github
<img width="1327" alt="Screenshot 2024-01-12 at 12 41 02 PM" src="https://github.com/razorpay/blade/assets/24487274/d93649e5-54c5-4f32-840b-c00101bd91ea">